api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -856,9 +853,11 @@ public DbUser syncTwoFactorAuthStatus(DbUser targetUser, Agent agent) {
             row ->
                 MeasurementBundle.builder()
                     .addMeasurement(GaugeMetric.USER_COUNT, row.getUserCount())
+                    // TODO remove.  until then, what we're interested in is registered vs not
                     .addTag(
                         MetricLabel.DATA_ACCESS_LEVEL,
-                        DbStorageEnums.dataAccessLevelFromStorage(row.getDataAccessLevel())
+                        AccessTierService.temporaryDataAccessLevelKluge(
```

Comment:
Since we don't store Data Access Level on the user anymore, we fake it with this kluge.  (Soon we will not report it at all.  This allows for a transition.)